src/dataframe/io.jl

```diff
@@ -517,7 +517,7 @@ function builddf(rows::Integer,
             values = Array(o.eltypes[j], rows)
         end
 
-        missing = falses(rows)
+        missing = fill(false, rows)
```

Comment:
I realize I missed that comment. That was because `NullableArray` stores missingness as a `Vector{Bool}`, not a `BitArray`.